editor/modes/visual-editor/block-list.js

```diff
@@ -213,6 +227,7 @@ class VisualEditorBlockList extends Component {
 							uid={ uid }
 							blockRef={ ( ref ) => this.setBlockRef( ref, uid ) }
 							onSelectionStart={ () => this.onSelectionStart( uid ) }
+							onSelectionExtension={ () => this.onSelectionExtension( uid ) }
```

Comment:
It's beside the object of this PR, but I'll share the note: we probably shouldn't be binding (and re-binding) these methods before passing them to `VisualEditorBlock` instances — they're all grown up and can do that themselves in the constructor, or just pass their `uid` explicitly. :)